### ISSUE TITLE
Access request for srbala

### ISSUE THREAD

**Issue by srbala** [adoptium/infrastructure#2167]
Access level:

- [x] Non-privileged
- [ ] jenkins user
- [ ] root/Administrative
- [ ] other (Please specify):

System for which access is needed:
x86_64, aarm64, ppc64le, s390x

Please explain why you need this access including whether it is a temporary or permanent request: 
Working on a PR fix for slim package, which affects multiple systems. https://github.com/AdoptOpenJDK/openjdk-docker/issues/339 team member recommented to access for my unit testing

**Comment by Willsparker** [comment]
(@sxa is on leave for a bit, so I'm picking this up). 

`test-packet-ubuntu1604-x64-1` : 147.75.204.239
`build-alibaba-ubuntu1804-armv8-1` : 119.8.164.219
`build-osuosl-centos74-ppc64le-1` : 140.211.168.138
`build-marist-rhel77-s390x-1` : 148.100.86.102

Will these machines suffice? They all have docker installed and a fair bit of space free. If you send me your public ssh key I can create a user for you on each machine :-)

**Comment by srbala** [comment]
https://raw.githubusercontent.com/srbala/srbala/master/keys/srbala.pub

**Comment by Willsparker** [comment]
`srbala@<IP_Address>` should work now. I assume this is a temporary request, so I'll keep the issue open until you're finished.

**Comment by srbala** [comment]
@Willsparker Thank you. I have one PR Open waiting on review. Planning to make another one soon,  you can close afer it.

**Comment by Willsparker** [comment]
@srbala Have your PRs been reviewed / merged yet? :-)

**Comment by srbala** [comment]
@Willsparker one waiting to be reviewed, working on another PR

**Comment by sxa** [comment]
ping @srbala - is the access still required? If still awaiting other PRs can you link to them here please so we can track?

**Comment by srbala** [comment]
My access can be removed. 

I have another PR for review, its for only X86 https://github.com/AdoptOpenJDK/openjdk-docker/pull/592

**Comment by sxa** [comment]
Thanks for the confirmation :-)
